app/src/main/java/com/github/onlynotesswent/model/folder/FolderViewModel.kt

```diff
@@ -109,7 +109,7 @@ class FolderViewModel(private val repository: FolderRepository) : ViewModel() {
       folder: Folder,
       onSuccess: () -> Unit = {},
       onFailure: (Exception) -> Unit = {},
-      isDeckView: Boolean = false,
+      isDeckView: Boolean,
```

Comment:
Nice removing the default value will ensure we don't accidentally get non deck folder when we are in deck view